source/manual/covid-19-services.html.md

```diff
@@ -0,0 +1,309 @@
+technical documentation][paas-docs].
+
+#### Get access
+
+To get access, ask an organisation manager to invite you to the
+`govuk_development` organisation.  They need to give you a role that
+lets you access both `staging` and `production` spaces and the
+applications inside.
+
+#### Log in
+
+Before you start, [install the CloudFoundry CLI][cf-cli].
+
+1. Log in with `cf login -a api.cloud.service.gov.uk --sso`
+2. Select staging or production with `cf target -s <staging or production>`
+
+Both applications are Rails applications, and use the
+[`ruby_buildpack`][] to build and deploy to PaaS.  You can read more
+about [managing PaaS applications][paas-managing] and [monitoring PaaS
+applications][paas-monitoring].
+
+#### Manage backing services
+
+The business volunteer application uses PaaS's [Redis backing
+service][paas-redis].  The backing service is attached to the
+application by a setting in the `manifest.yml` file in the repository
+root.
+
+The JSON data in the `VCAP_SERVICES` environment variable exposes
+settings and credentials for the backing service automatically.
+
+### AWS
+
+We have two AWS accounts (staging and production) for CDN and data
+storage. They contain DynamoDB, IAM and CloudFront resources,
+which were provisioned by Terraform in the [tech-ops-private
+repository](techops-repo).
+
+To log in to AWS:
+
+1. [Install and set up the gds-cli](/manual/get-started.html#8-use-your-aws-access), then log in to the AWS Console with either:
+
+   - `gds aws govuk-corona-data-staging -l` for staging
+   - `gds aws govuk-corona-data-prod -l` for production
+
+
+## Deployment
+
+Deployment is managed via [TechOps shared Concourse][big-concourse].
+CI/CD pipelines are configured under the `govuk-tools` team.  Any
+changes to application code on the master branch are continuously
+deployed to staging.  Smoke tests are set up to run on staging, if
+these pass the applications are automatically deployed to production.
+The smoke tests are the feature tests within the application
+repository.
+
+The Concourse deployment pipeline and task configuration for each
+application is stored in the "concourse" directory at the root of each
+repository.
+
+Links to Concourse Pipelines:
+
+- [`govuk-corona-business-volunteer-form`](https://cd.gds-reliability.engineering/teams/govuk-tools/pipelines/govuk-corona-business-volunteer-form)
+- [`govuk-corona-vulnerable-people-form`](https://cd.gds-reliability.engineering/teams/govuk-tools/pipelines/govuk-corona-vulnerable-people-form)
+
+For access login with your GitHub account, you need to be a part of
+the GOV.UK Production team.  You'll also need to be on the VPN.
+
+### Administering Pipelines
+
+To administer pipelines you must have the `pipeline-operator` role.
+Pipelines are administered using the `fly` CLI tool.  `fly` can be
+downloaded directly from the Concourse dashboard using the links in
+the bottom bottom right of the screen (you probably want to click the
+apple).
```

Comment:
For me, `fly` downloads as a text file.

Reply:
You can `brew cask install fly`, too.  Thanks to @benthorner for finding that one!

```suggestion
apple). If that doesn't download an executable file, you can install with `brew cask install fly`.
```